<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test</class>
 <widget class="QWidget" name="test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>test</string>
  </property>
  <widget class="QPushButton" name="csvSearchButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>.csv-Datei auswÃ¤hlen</string>
   </property>
  </widget>
  <widget class="QLabel" name="pathLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pfad</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="pathText">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>241</width>
     <height>70</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="resultsButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Datei auswerten</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
